tensorflow/lite/micro/examples/recipes/BUILD

```diff
@@ -8,6 +8,7 @@ py_library(
     name = "resource_variables_lib",
     srcs = ["resource_variables_lib.py"],
     srcs_version = "PY3",
+    visibility = ["//visibility:public"],
```

Comment:
can we make this ["//:__subpackages__"] ?

Reply:
Done.